package UserExceptionDefine;

public class Throws2 {
    public static void main(String[] args) throws InterruptedException {
        System.out.println("프로그램 시작");
        Thread.sleep(3000);
        System.out.println("프로그램 종료");
    }
}
